sdk/dotnet/Authentication/V1Beta1/TokenReview.cs

```diff
@@ -77,10 +77,6 @@ private static CustomResourceOptions MakeResourceOptions(CustomResourceOptions?
             var defaultOptions = new CustomResourceOptions
```

Comment:
@lblackstone I believe this solving this would require some engineering lift after investigating how we're doing SDK and schema generation. I'm looking at another resource we removed much earlier, `ComponentStatus`, and they are also still generated in the SDK code, but empty:
https://github.com/pulumi/pulumi-kubernetes/blob/7b1bd5aecff6bc2a6ebe9d1934a6585a0db27104/sdk/dotnet/Core/V1/Outputs/ComponentStatus.cs#L14

Reply:
It should be fixed now. I had to manually delete the `authentication` and `authorization` directories from each SDK before regenerating them, and that fixed it.